Formula/cadence.rb

```diff
@@ -21,10 +21,10 @@ def install
 
   test do
     (testpath/"hello.cdc").write <<~EOS
-      pub fun main() {
-        log("Hello, world!")
+      pub fun main(): Int {
+        return 0
       }
     EOS
-    assert_match "Hello, world!", shell_output("#{bin}/cadence hello.cdc")
+    system "#{bin}/cadence", "hello.cdc"
```

Comment:
can we assert the output?

Reply:
I took out the `log` call, so there's no output to assert. I've opened an issue with upstream to advise on why `log` is causing the panic: https://github.com/onflow/cadence/issues/361

Reply:
👍